ui-v2/app/styles/components/composite-row/skin.scss

```diff
@@ -6,9 +6,9 @@
   border-left-color: $transparent;
 }
 %composite-row-intent {
-  border-color: $gray-200;
+  border-color: $gray-200 !important;
   box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-  border-top-color: $transparent;
+  border-top-color: $transparent !important;
```

Comment:
Are these `!important`s definitely need here?

Reply:
Not anymore 🤔 I couldn't get the styling to work, but it must have been early on. Removed the !important.